mindmaps-engine/src/main/java/io/mindmaps/engine/loader/BlockingLoader.java

```diff
@@ -123,6 +122,9 @@ private void loadData(String name, Collection<Var> batch) {
                     handleError(e, 1);
                 }
             }
+        } catch (Throwable e){
+            LOG.error(e.getMessage());
```

Comment:
You can just to `LOG.error(ErrorMessage.FAILED_TRANSACTION.getMessage(repeatCommits), e)` . No need for two log lines really.